homepage.css

```diff
@@ -144,54 +155,212 @@ ul li:hover ul{
 	height:214px;
 	background-color:white;
 	margin-top:50px;
+	position: relative;
+    width: 100%;
+}
+
+.images1 img{
+	height:214px;
+	margin-left:25px;
 }
 
 .images2{
 	height:214px;
 	background-color:white;
 	margin-top:50px;
+	position: relative;
+    width: 100%;
+}
+
+.images2 img{
+	height:214px;
+	margin-left:25px;	
 }
 
 .images3{
 	height:214px;
 	background-color:white;
 	margin-top:50px;
+	position: relative;
+    width: 100%;
+}
+
+.images3 img{
+	height:214px;
+	margin-left:25px;
 }
 
 .images4{
 	height:214px;
 	background-color:white;
 	margin-top:50px;
-
+	position: relative;
+    width: 100%;
 }
 
+.images4 img{
+	height:214px;
+	margin-left:25px;
+}
 
 .images5{
 	height:214px;
 	background-color:white;
 	margin-top:50px;
+	position: relative;
+    width: 100%;
+}
+
+.images5 img{
+	height:214px;
+	margin-left:25px;
 }
 
 .images6{
 	height:214px;
 	background-color:white;
 	margin-top:50px;
+	position: relative;
+    width: 100%;
+}
+
+.images6 img{
+	height:214px;
+	margin-left:25px;
 }
 
 .images7{
 	height:214px;
 	background-color:white;
 	margin-top:50px;
+	position: relative;
+    width: 100%;
+}
+
+.images7 img{
+	height:214px;
+	margin-left:25px;
 }
 
 .images8{
 	height:214px;
 	background-color:white;
 	margin-top:50px;
-	
+	position: relative;
+    width: 100%;
 }
 
-.content_4 img{
+.images8 img{
+	height:214px;
 	margin-left:25px;
-	border: 2px solid black;
+}
+
+
+
+.image1 {
+  display: block;
+  width: 100%;
+  height: auto;
+}
+
+.image2 {
+  display: block;
+  width: 100%;
+  height: auto;
+}
+
+.image3 {
+  display: block;
+  width: 100%;
+  height: auto;
+}
+
+.image4 {
+  display: block;
+  width: 100%;
+  height: auto;
+}
+
+.image5 {
+  display: block;
+  width: 100%;
+  height: auto;
+}
+
+.image6 {
+  display: block;
+  width: 100%;
+  height: auto;
+}
+
+.image7 {
+  display: block;
+  width: 100%;
+  height: auto;
+}
+
+.image8 {
+  display: block;
+  width: 100%;
+  height: auto;
+}
+
+
+.overlay {
+	position: absolute;
+	top: 0;
+	bottom: 0;
+	left: 0;
+	right: 0;
+	height: 100%;
+	width: 100%;
+	opacity: 0;
+	transition: .5s ease;
+	background-color: black;
+	margin-left:25px;
+	font-family: 'Open Sans', sans-serif;
+	font-weight:300;
+	text-decoration:none;
+	color:white;
+}
+
+.images1:hover .overlay {
+	opacity: 1;
+	
+}
+
+.images2:hover .overlay {
+  opacity: 1;
+}
+
+.images3:hover .overlay {
+  opacity: 1;
+}
+
+.images4:hover .overlay {
+  opacity: 1;
+}
+
+.images5:hover .overlay {
+  opacity: 1;
+}
+
+.images6:hover .overlay {
+  opacity: 1;
+}
+.images7:hover .overlay {
+  opacity: 1;
+}
+.images8:hover .overlay {
+  opacity: 1;
+}
+.text {
```

Comment:
31/08/2021: This will set the background when hover and the position of the text.(font size)